action.php

```diff
@@ -420,16 +478,16 @@ protected function generatePDF($cachefile) {
         $no_pages = count($this->list);
         foreach($this->list as $page) {
             $counter++;
-            $filename = wikiFN($page, $REV);
-
-            if(!file_exists($filename)) {
-                continue;
-            }
 
             // set global pageid to the rendered page
             $ID = $page;
 
-            $pagehtml = p_cached_output($filename, 'dw2pdf', $page);
+            //$pagehtml = p_cached_output($filename, 'dw2pdf', $page);
```

Comment:
Is it fine to remove the commented code?

Reply:
Yes, you are right